x-pack/solutions/observability/plugins/synthetics/e2e/synthetics/journeys/alert_rules/custom_tls_alert.journey.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { journey, step, before, after, expect } from '@elastic/synthetics';
+import { syntheticsAppPageProvider } from '../../page_objects/synthetics_app';
+import { SyntheticsServices } from '../services/synthetics_services';
+
+journey(`CustomTLSAlert`, async ({ page, params }) => {
+  const syntheticsApp = syntheticsAppPageProvider({ page, kibanaUrl: params.kibanaUrl, params });
```

Comment:
we should also try to interact with filters in the journey to make sure they continue to work

Reply:
I've added interaction with the KQL filter bar and with the type filter in my e2e tests.